### PR TITLE
feat(EMS-1694-1695): Account reactivation - send email with reactivation link

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds the initial user flow for reactivating a blocked account.

## Changes
- Add `reactivationHash` and `reactivationExpiry` fields to accounts in the API schema and DB.
- Update account sign in GQL resolver to return `accountId`.
- Create new GQL resolver `sendEmailReactivateAccountLink`. This resolver:
  - Checks if an account exists.
  - Generate a reactivation hash/token.
  - Update the account with the new hash/token.
  - Send an email to the account's email addresss with the hash/token.
- Add new email function to send the "reactivate account" email.
- Setup UI route, controller and template for the "reactivate your account - email sent" page.
- Add  POST route to the "account blocked" page so that we can redirect to the "reactivate your account" page with the account ID in the query params.
- Update the "customer service helpline" component to _not_ render a heading. The heading now differs between different pages, so each page that consumes this component now renders their own heading:
  - Manage account.
  - Contact us.
  - Reactivate account - email sent.
- Fixed/updated account suspension routes to have `/insurance`.

## Other improvements
- Add documentation to `DATE_30_MINUTES_FROM_NOW` API constant.
- Fixed some typos.

### CODE REVIEW

File: e2e-tests/constants/routes/insurance/account.js [hunk 1]
```diff
@@ -36,8 +36,13 @@ const SIGNED_OUT = `${INSURANCE_ROOT}/signed-out`;
 
 const SUSPENDED = {
   ROOT: SUSPENDED_ROOT,
+  EMAIL_SENT: `${SUSPENDED_ROOT}/email-sent`,
+  VERIFY_EMAIL: `${SUSPENDED_ROOT}/verify-email`,
+  VERIFY_EMAIL_LINK_EXPIRED: `${SUSPENDED_ROOT}/link-expired`,
 };
 
+const REACTIVATED = `${INSURANCE_ROOT}/account-reactivated`;
```
Comment (Zainzzkk) [inline e2e-tests/constants/routes/insurance/account.js:44]: can we keep this on the top with all the string consts? not a blocker

Comment (ttbarnes) [reply e2e-tests/constants/routes/insurance/account.js:44]: Good shout, working on this route for the next PR, have already updated 👍

File: src/api/custom-resolvers/mutations/send-email-reactivate-account-link.test.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+const {
+  ENCRYPTION: {
+    PASSWORD: {
+      PBKDF2: { KEY_LENGTH },
+    },
+  },
+  MAX_PASSWORD_RESET_TRIES,
+} = ACCOUNT;
+
+describe('custom-resolvers/send-email-password-reset-link', () => {
```
Comment (Zainzzkk) [inline src/api/custom-resolvers/mutations/send-email-reactivate-account-link.test.js:30]: wrong description?

Comment (ttbarnes) [reply src/api/custom-resolvers/mutations/send-email-reactivate-account-link.test.js:30]: Good spot this is actually running the wrong test as well, whoops. Fixing

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**